mlflow/server/js/.eslintrc.js

```diff
@@ -585,26 +585,13 @@ module.exports = {
         'no-shadow': 'off',
         '@typescript-eslint/no-shadow': 'off',
 
-        // Please leave this rule as the last item in the array, as it's quite large
-        '@typescript-eslint/ban-types': [
```

Comment:
`@typescript-eslint/ban-types` is deprecated.

https://typescript-eslint.io/rules/ban-types/